regolith/schemas.py

```diff
@@ -3620,6 +3621,172 @@
             "type": "boolean",
         },
     },
+    "projecta": {
+        "_description": {
+            "description": "This collection describes a single deliverable "
+                            "of a larger project."
+        },
+        "_id": {
+            "description": "Unique projecta identifier",
+            "required": True,
+            "type": "string"
+        },
+        "begin_date": {
+            "description": "projecta start date, yyyy-mm-dd",
+            "required": False,
+            "type": "string"
+        },
+        "collaborators": {
+            "description": "list of collaborator's id's",
+            "required": False,
+            "type": "list"
+        },
+        "deliverable":{
+            "description": "outline of the deliverable for this projecta",
+            "type": "dict",
+            "schema": {
+                "audience": {"description": "who is this deliverable for",
+                             "required": False,
+                             "type": "list"},
+                #is this the proper form of the due date, the exemplar is ambiguous
+                "due_date": {"description": "due date of deliverable, yyyy-mm-dd",
+                             "required": False,
+                             "anyof_type": ["date", "string"]},
+                "success_def": {"description": "definition of a successful deliverable",
+                                "required": False,
+                                "type": "string"},
+                "scope": {"description": "",
+                          "required": False,
+                          "type": "list"},
+                "platform": {"description": "description of how and where the audience will access "
```

Comment:
nicely done.  Please add at the end "For software releases, this may be the computer operating systems that will be supported, or if it will be a web service, etc."